Source/Web.Maple.Client/Web.Maple.Client.csproj

```diff
@@ -6,7 +6,7 @@
     <Nullable>enable</Nullable>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <Authors>Wilderness Labs, Inc</Authors>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
```

Comment:
ok this is an issue if we want to use the client library on Meadow. If we're only targetting other platforms then no-problem. I'd personally vote to keep it netstandard2.0.

Reply:
good catch, I'll retarget netstandard2.1 like we do for all our meadow devices.